### ISSUE TITLE
Map models are only valid for the host when overriding map objects

### ISSUE THREAD

**Issue by trende2001** [Facepunch/sbox-issues#6781]
### Describe the bug

When I'm overriding a map's entities, like a door for example, the models always end up missing for other players except the host.

I networkspawn the doors so other clients can see them.

### To Reproduce

1. Clone https://github.com/Softsplit/sandbox
2. Open the scene 'game'
3. Start it and join with another instance
4. Find a door on the map
5. On the host it's there, on the client it isn't

### Expected behavior

Map models should exist for the clients.
![Image](https://github.com/user-attachments/assets/62334b65-7a7d-4996-b37d-b89a6070d0d2)

### Host:
![Image](https://github.com/user-attachments/assets/188508de-c36b-4ba5-8174-f0e0fec189b2)

### Client:
![Image](https://github.com/user-attachments/assets/ac2246cf-9438-4148-8644-1558eb3328c5)
![Image](https://github.com/user-attachments/assets/897c791b-aa7b-48e6-90ee-e254efb32790)

### Additional context

The door component is being created here:
https://github.com/Softsplit/sandbox/blob/cfd87d93f20e000239438ceba34d8a18cd578d77/Code/Components/CustomMapInstance.cs#L8

Door component itself:
https://github.com/Softsplit/sandbox/blob/main/Code/Components/Map/Door.cs

**Comment by handsomematt** [comment]
Networked door gameobject is probably deserializing before the mapinstance mounts the vpk.. Maybe a priority hack can be done, but I don't think we want MapInstance to be the way people are doing things in the end anyway

**Comment by trende2001** [comment]
> Networked door gameobject is probably deserializing before the mapinstance mounts the vpk.. Maybe a priority hack can be done, but I don't think we want MapInstance to be the way people are doing things in the end anyway

Yeah, I think a stop-gap solution for now would be good until they're phased out

**Comment by LucasLy** [comment]
just get better buddy
